frontend/src/main/web/src/app/pages/admin/dashboards/trouble-statistics/trouble-statistics.component.ts

```diff
@@ -1,86 +1,99 @@
-import { Component, OnInit } from '@angular/core';
-import * as CanvasJS from '../../../../../assets/js/canvasjs.min';
-import { TroubleStatisticsService } from '../trouble-statistics.service';
-import { TroubleStatisticsModel } from './trouble-statistics.model';
-import { FormGroup, FormControl } from '@angular/forms';
+import {Component, OnInit} from '@angular/core';
+import * as CanvasJS from '../../canvasjs.min';
+import {TroubleStatisticsService} from '../trouble-statistics.service';
+import {TroubleStatisticsModel} from './trouble-statistics.model';
+import {FormControl, FormGroup} from '@angular/forms';
 
 @Component({
-  selector: 'app-trouble-statistics',
-  templateUrl: './trouble-statistics.component.html',
-  styleUrls: ['./trouble-statistics.component.scss']
-})
+             selector: 'app-trouble-statistics',
+             templateUrl: './trouble-statistics.component.html',
+             styleUrls: ['./trouble-statistics.component.scss']
+           })
 export class TroubleStatisticsComponent implements OnInit {
   troubleStatisticsModel: TroubleStatisticsModel;
+
   ticketChart: CanvasJS.Chart;
 
   form = new FormGroup({
-    isTotal: new FormControl('total'),
-    id: new FormControl(''),
-  });
+                         isTotal: new FormControl('total'),
+                         id: new FormControl(''),
+                       });
 
-  constructor(private troubleStatisticsService: TroubleStatisticsService)
-  {
+  constructor(private troubleStatisticsService: TroubleStatisticsService) {
   }
 
   onSubmit() {
-    if(this.form.value.isTotal == "true" || this.form.value.id == "")
-    {
-      this.troubleStatisticsService.getStatistic().subscribe(data => {
-        this.troubleStatisticsModel = data;
-        this.ticketChart.options.title.text = "Tickets:";
-        this.reloadChart();
-      });
+    if (this.form.value.isTotal == "true" || this.form.value.id == "") {
+      this.troubleStatisticsService.getStatistic()
+          .subscribe(data => {
+            this.troubleStatisticsModel = data;
+            this.ticketChart.options.title.text = "Tickets:";
+            this.reloadChart();
+          });
     }
-    else
-    {
-      this.troubleStatisticsService.getStatisticForApprover(this.form.value.id).subscribe(data => {
-        this.troubleStatisticsModel = data;
-        this.ticketChart.options.title.text = "Tickets for approver id " + this.form.value.id + ":";
-        this.reloadChart();
-      });
+    else {
+      this.troubleStatisticsService.getStatisticForApprover(this.form.value.id)
+          .subscribe(data => {
+            this.troubleStatisticsModel = data;
+            this.ticketChart.options.title.text = "Tickets for approver id " + this.form.value.id + ":";
+            this.reloadChart();
+          });
     }
   }
 
-  reloadChart()
-  {
+  reloadChart() {
     this.ticketChart.options.data[0].dataPoints = [];
     this.ticketChart.options.data[0].dataPoints.push(
```

Comment:
This code and the code below has one behavior. It would be better to extract it in the function, which accepts arguments for the label and y point

Reply:
it's Roma's component 
I think it would be better if he change it by himself 
But if it's necessary, I can do it as well

Reply:
If you don't mind, please refactor code. Instead of ask Roma to make a branch and create a new pull request which will take more time

Reply:
Done